website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -0,0 +1,758 @@
+* `permissions_boundary_arn` - Permission Boundary Arn (Policy) for NGAP compliance
+* `postgres_user_pw` - password for the postgres user
+* `database_name` - orca
+* `database_app_user` - orca_user
+* `database_app_user_pw` - the password for the application user
+
+**Optional:**
+* `prefix` - Prefix that will be pre-pended to resource names created by terraform.
+  Defaults to `dr`.
+* `profile` - AWS CLI Profile (configured via `aws configure`) to use.
+  Defaults to `default`.
+* `region` - Your AWS region.
+  Defaults to `us-west-2`.
+* `restore_expire_days` - How many days to restore a file for.
+  Defaults to 5.
+* `restore_request_retries` - How many times to retry a restore request to Glacier.
+  Defaults to 3.
+* `restore_retry_sleep_secs` - How many seconds to wait between retry calls to `restore_object`.
+  Defaults to 3.
+* `restore_retrieval_type` -  the Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'.
+  Defaults to `Standard`. Understand the costs associated with the tiers before modifying.
+* `copy_retries` - How many times to retry a copy request from the restore location to the archive location.
+  Defaults to 3.
+* `copy_retry_sleep_secs` - How many seconds to wait between retry calls to `copy_object`.
+  Defaults to 0.
+* `ddl_dir` - the location of the ddl dir that contains the sql to create the application database.
+  Defaults to 'ddl/'.
+* `drop_database` - Whether to drop the database if it exists (True), or keep it (False).
+  Defaults to False.
+* `database_port` - the port for the postgres database.
+  Defaults to '5432'.
+* `platform` - indicates if running locally (onprem) or in AWS (AWS).
+  Defaults to 'AWS'.
+
+
+
+
+## Deploy ORCA with Terraform
+
+In the proper module directory, initialize and apply changes using the commands
+below.
+
+1. Run `terraform init`.
+2. Run `terraform plan` #optional, but allows you to preview the deploy.
+3. Run `terraform apply`.
+
+This commands above will create and deploy ORCA. To delete the created objects,
+run `terraform destroy`.
+
+
+## Collection Configuration
+
+To configure a collection to enable ORCA, add the line
+`"granuleRecoveryWorkflow": "DrRecoveryWorkflow"` to the collection configuration
+as seen below. Optionally, you can exclude files by adding values to an
+`"excludeFfileType"` variable. For more information, see the documentation on the
+[`copy_to_glacier` task](https://github.com/nasa/cumulus-orca/tree/master/tasks/copy_to_glacier).
+
+```json
+{
+  "queriedAt": "2019-11-07T22:49:46.842Z",
+  "name": "L0A_HR_RAW",
+  "version": "1",
+  "sampleFileName": "L0A_HR_RAW_product_0001-of-0420.h5",
+  "dataType": "L0A_HR_RAW",
+  "granuleIdExtraction": "^(.*)((\\.cmr\\.json)|(\\.iso\\.xml)|(\\.tar\\.gz)|(\\.h5)|(\\.h5\\.mp))$",
+  "reportToEms": true,
+  "granuleId": "^.*$",
+  "provider_path": "L0A_HR_RAW/",
+  "meta": {
+    "granuleRecoveryWorkflow": "DrRecoveryWorkflow",
+    "exclue_file_type": [".cmr", ".xml", ".met"]
```

Comment:
Change `exclue_file_type` to `excludeFileTypes`.

Reply:
Made update to excludeFileTypes